picachu/config/s3_config.py

```diff
@@ -6,12 +6,15 @@
 s3_bucket_name = os.getenv('S3_BUCKET_NAME')
 s3_prefix = os.getenv('S3_PREFIX', default='')
 s3_use_ssl = os.getenv('S3_USE_SSL', default='true').lower()
+s3_external = os.getenv('S3_EXTERNAL')
```

Comment:
Let's still remove this functionality with s3 and add a new variable. I will then create a separate card for it for myself, we are currently discussing with devops how best to do this.